rust-src/ffi_helpers/src/ffi_macros.rs

```diff
@@ -169,7 +169,8 @@ macro_rules! macro_free_ffi {
                 return;
             }
             unsafe {
-                $fr(ptr);
+                let v = $fr(ptr);
+                drop(v);
```

Comment:
Clippy is a bit pedantic about this since some recent version.
`Box::from_raw` is declared as `must_use` so we must use the result.

It also has a nice explicitness to it, to call drop. But of course we could just write
```
let _ = $fr(ptr);
```